### PR TITLE
deps: use dependencyGraphBuilder.buildFullDependencyGraph

### PR DESCRIPTION
**Pull request by suztomo**
@stephaniewang526 @elharo 
CC: @yangnuoyu 

I wanted to see whether we need a release of `com.google.cloud.tools:dependencies`. It does not seem needing a release.

With dependencyGraphBuilder.buildFullDependencyGraph, which is marked public in `com.google.cloud.tools:dependencies:1.4.3`. This "dashboard" module no longer needs the copy of the Java code in "com.google.cloud.tools.opensource.cloudbomdashboard.dependencies" package. (and thus this PR deletes the package)

### CODE REVIEW

**Comment by yangnuoyu** [comment]
I will agree with you if "buildFullDependencyGraph(ImmutableList.of(artifact))" can replace the "buildVerboseDependencyGraph(artifact)". I was not sure about this when I built the dashboard.

**Comment by elharo** [comment]
buildFull and buildVerbose are not the same. You should pick the one that fits the use case.

**Comment by suztomo** [comment]
For this dashboard, I confirmed that both buildFullDependencyGraph and buildVerboseDependencyGraph generate the same result; 5 in the "upper bound errors" and 16 in "fail to converge" section.

Screen Shot 2020-07-29 at 09 44 52


The difference of the two methods are whether to include optional dependencies: buildFullDependencyGraph includes them and buildVerboseDependencyGraph does not.

**Comment by stephaniewang526** [comment]
LGTM - looks like we can go w buildFull instead here. Pls keep us posted on release so @yangnuoyu could refactor the dashboard code.

**Comment by suztomo** [comment]
@stephaniewang526 PTAL; I fixed the commit message to follow combentionalcommits.

File: dashboard/src/main/java/com/google/cloud/tools/opensource/cloudbomdashboard/DashboardMain.java [hunk 1]
```diff
@@ -44,8 +41,16 @@
 import java.time.LocalDateTime;
 import java.util.*;
```
Comment (elharo) [inline dashboard/src/main/java/com/google/cloud/tools/opensource/cloudbomdashboard/DashboardMain.java:42]: we shouldn't be using wildcard imports

Comment (suztomo) [reply dashboard/src/main/java/com/google/cloud/tools/opensource/cloudbomdashboard/DashboardMain.java]: Fixed.

**Comment by suztomo** [comment]
@stephaniewang526 @elharo I don't have "write access" to this repository. Would you merge this?

**Comment by suztomo** [comment]
Merged origin/master. I'm going to release "dependencies" module later 1.5.0 this week. But this PR does not need to wait for it.

**Comment by suztomo** [comment]
@stephaniewang526 Thank you.

**Comment by suztomo** [comment]
@yangnuoyu @stephaniewang526 We released the dependencies module 1.5.0 today:
https://search.maven.org/artifact/com.google.cloud.tools/dependencies/1.5.0/jar

**Pull request merged**